articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -9,30 +9,369 @@ beta: true
+:::
+
+### Implement the Mobile App
+
+In this section we will see how we can implement a Mobile application for our scenario.
+
+::: note
+[See the implementation in Android at the bottom of this document](/architecture-scenarios/application/mobile-api/mobile-implementation-android#1-set-up-the-application)
+:::
+
+#### Authorize the User
+
+To authorize the user we will implement an [Authorization Code Grant Flow with PKCE](/api-auth/tutorials/authorization-code-grant-pkce). The mobile application should first send the user to the [authorization URL](/api/authentication#authorization-code-grant-pkce-) along with the `code_challenge` and the method used to generate it:
+
+```
+https://YOUR_AUTH0_DOMAIN/authorize?
+    audience=API_AUDIENCE&
+    scope=SCOPE&
+    response_type=code&
+    client_id=YOUR_CLIENT_ID&
+    code_challenge=CODE_CHALLENGE&
+    code_challenge_method=S256&
+    redirect_uri=https://YOUR_APP/callback
+```
+
+The `GET` request to the authorization URL should include the following values:
+
+- __domain__: The value of your Auth0 Domain. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](https://manage.auth0.com/#/clients).
+- __client_id__: The value of your Auth0 Client Id. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](https://manage.auth0.com/#/clients).
+- __audience__: The value of your API Identifier. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](https://manage.auth0.com/#/clients).
+- __scope__: The [scopes](/scopes) which determine the information to be returned in the `id_token` and `access_token`. A scope of `openid profile email offline_access` will return all the user profile information the `id_token` and return a `refresh_token`. You also need to request the scopes required to call the API, in this case the `read:timesheets create:timesheets` scopes. This will ensure that the `access_token` has these scopes.
+- __response_type__: Indicates the Authentication Flow to use. For a Mobile application using PKCE, this should be set to `code`.
+- __code_challenge__: The generated code challenge from the code verifier. You can find instructions on generating a code challenge [here](/api-auth/tutorials/authorization-code-grant-pkce#1-create-a-code-verifier).
+- __code_challenge_method__: Method used to generate the challenge. Auth0 supports only `S256`.
+- __redirect_uri__: The URL which Auth0 will redirect the browser to after authorization has been granted by the user. The Authorization Code will be available in the code URL parameter. This URL must be specified as a valid callback URL under your [Client's Settings](https://manage.auth0.com/#/clients).
+
 ::: note
-This architecture scenario is under construction and will be updated soon.
+[See the implementation in Android at the bottom of this document](/architecture-scenarios/application/mobile-api/mobile-implementation-android#2-authorize-the-user)
```

Comment:
These notes need to change, the implementation is not at the bottom of this document (this applied only at the google doc)

Use `See the implementation in Android.`. See the other architecture scenario docs for details, for example https://github.com/auth0/docs/blob/master/articles/architecture-scenarios/application/server-api.md